core/lagompb-core/src/main/scala/io/superflat/lagompb/BaseServiceImpl.scala

```diff
@@ -228,49 +172,5 @@ trait BaseGrpcServiceImpl extends SharedBaseServiceImpl {
         case Success(value) => Success(value)
       }
 
-  /**
```

Comment:
We need a send command where the developer does not need to supply the entity ID because it has it as an annotation on the command proto definition. I cannot see anywhere where this is implemented.

Reply:
per convo, let's think harder about that and add it back in a subsequent release. it should be more DRY and reuse the existing sendCommand. Given nobody is using it yet, we will implement the better version soon.

Reply:
https://github.com/super-flat/lagom-pb/issues/171